CONTRIBUTING.md

```diff
@@ -26,6 +26,10 @@ The process for contributing looks like this:
    1. Follows the [guidelines](#guidelines)
    2. Meets the [technical requirements](#technical-requirements)
    3. Includes [OWNERS](#owning-and-maintaining-a-task)
+   4. Successfully passes tests using at least one of the methods listed below.
```

Comment:
I think this would be more readable without the examples; just link to the script.

Reply:
updated